mcaleb808/Taxi-24-APIs
# setup node/express project

---

#### Description of changes made

> - initialize the project
> - setup file structure
> - configure eslint and prettier
> - configure GitHub actions and a PR template
#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/mcaleb808/Taxi-24-APIs), and `cd into the project`
- Check out this branch by _git checkout `setup-project`_
- Run `yarn | npm install` to install the project dependencies
- Run `yarn dev | npm run dev` to start the local server.

#### Checklist:



#### Screenshots

|        BEFORE        |       AFTER       |
| :------------------: | :---------------: |
| Before these changes | After the changes |
